### DISCUSSION TITLE
de/2-learning-circle/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos#17] [General]
# de/2-learning-circle/

https://lernos.org/de/2-learning-circle/

**Comment by WolframStrauss** [comment]
Der Verweis https://lernos.org/de/guides/ von oben läuft ins Leere

**Reply by simondueckert** [reply]
Danke für den Hinweis, Link ist gefixt.

**Comment by WolframStrauss** [comment]
Hier ist mir der Begriff Lernpfad nicht klar, wird leider auch nicht im [Glossar](https://lernos.org/de/lernos-glossary/#l) erklärt. In eurer Vorlage (`lernOS-Circle-Template-de`, die ich leider nicht mehr finden kann. Verweis unter [Hilfsmittel](https://lernos.org/de/lernos-for-you/#hilfsmittel) würde helfen) scheint es drei Pfade zu geben:
- Getting Things Done (GTD)
- Objectives & Key Results (OKR)
- Working Out Loud (WOL)
Sind das die drei Pfade, oder habe ich das falsch verstanden?
Danke & Grüße. Wolfram

**Reply by simondueckert** [reply]
Das ist korrekt, es gibt drei Lernpfade, wie in der Übersicht dargestellt: https://cogneon.github.io/lernos-for-you/de/2-0-Lernpfade/ - jeder ist für 13 Wochen bei 1-2h Zeit pro Woche angelegt.
